wemake_python_styleguide/constants.py

```diff
@@ -309,6 +309,19 @@
 #: Maximum length of ``yield`` ``tuple`` expressions.
 MAX_LEN_YIELD_TUPLE: Final = 5
 
+#: Approximate constants which real values should be imported from math module.
+APPROXIMATE_CONSTANTS: Final = frozenset((
+    3.14,
```

Comment:
Sorry, that's not it should be done. We need to store real constants in this set. And then check that your own numbers are almost equal to these constants.

Reply:
Tried to use round, but pi rounded to 3.1416 and 3.142 because full form is 3.14159. What should we do with this?